lib/components/map_widget.dart

```diff
@@ -12,9 +12,8 @@ import 'package:location_permissions/location_permissions.dart';
 
 class MapWidget extends StatefulWidget {
   final bool isStatic;
-  final VoidCallback onMapReady;
 
-  MapWidget({Key key, @required this.isStatic, this.onMapReady})
```

Comment:
seems linke onMapReady is not needed anymore?!

Reply:
done